MemoTesi/main.tex

```diff
@@ -0,0 +1,341 @@
+\par
+\noindent
+\begin{minipage}[t]{0.04\textwidth}
+~
+\end{minipage}
+\begin{minipage}[t]{0.4\textwidth}
+{\large{\bf Relatore:\\
+Chiar.mo Prof.\\
+RELATORE}}
+\end{minipage}
+\hfill
+\begin{minipage}[t]{0.4\textwidth}\raggedleft
+{\large{\bf Presentata da:\\
+TU}}
+\end{minipage}
+\begin{minipage}[t]{0.04\textwidth}
+~
+\end{minipage}
+
+\vspace{30mm}
+
+\begin{center}
+    {\large{\bf \rom{1} Sessione\\
+    Anno Accademico 2022/2023 }}
+\end{center}
+\end{titlepage}
+
+\restoregeometry
+\newpage
+\begin{center}
+    (DA FARE ALLA FINE)\\
+    5 parole chiave per caratterizzare il contenuto della dissertazione:\\ (se non ti piacciono così sparse puoi anche semplicamente scriverle su una riga sola)
+\end{center}
+
+% https://tex.stackexchange.com/questions/26538/words-scattered-randomly-in-on-coverpage
+\begin{tikzpicture}[overlay,remember picture,shift=(current page.center)]
+\pgfmathsetseed{3}
+
+
+\foreach [count=\count] \word in {Parola 1, parola 2, parola 3, parola 4, parola 5} {
+\node [
+    xshift={(mod(\count,3)-1)*(\paperwidth/4)},
+    yshift={(mod(\count,7)-3)*(\paperwidth/6)},
+    xshift=rand*4cm,
+    yshift=rand*2cm,
+    % rotate=rand*35,
+    % opacity=rnd*0.5+0.125,
+    font=\large] {\word};
+}
+\end{tikzpicture}
+\newpage
+
+\topmargin=6.5cm
+\begin{flushright}
+\emph{
+\LARGE{La dedica}\\\vspace{2mm}
+\LARGE{anche quella se vuoi}\\\vspace{3mm} 
+\LARGE{su più righe} 
+}
+\end{flushright}
+\newpage~\newpage
+\pagenumbering{gobble}
+\chapter*{Abstract}
+Abstract qui (ti consiglio di farlo alla fine)
+
+\topmargin=-1cm
+\tableofcontents
+\thispagestyle{empty}
+\listoftables
+\thispagestyle{empty}
+\listoffigures
+\thispagestyle{empty}
+\newpage~\newpage
+
+
+\pagenumbering{arabic}
+\setcounter{chapter}{-1}
+\raggedbottom
+\chapter{INTRODUZIONE} \label{chap:intro}
+\pagestyle{plain}
+\setcounter{page}{1}
+
+(Io l'introduzione l'ho scritta alla fine)\\
+bla bla bla
+\section{elenchi}
+\subsection{Elenchi puntati}
+\begin{itemize}
+    \item bla
+    \begin{itemize}
+        \item sub-bla
+    \end{itemize}
+    \item bla
+\end{itemize}
+
+\subsection{Elenchi numerati}
+\begin{enumerate}
+    \item bla1
+    \begin{enumerate}
+        \item sub bla 1
+        \item sub bla 2
+    \end{enumerate}
+    \item bla 2
+\end{enumerate}
+
+\subsection{Mix}
+\begin{itemize}
+    \item bla
+    \begin{enumerate}
+        \item sub bla 1
+    \end{enumerate}
+    \item bla
+\end{itemize}
+
+\begin{enumerate}
+    \item bla 1
+    \begin{itemize}
+        \item sub bla
+    \end{itemize}
+    \item bla 2
+\end{enumerate}
+
+\section{Font}
+\textbf{bla bla bla}\\
+\textit{Ancora bla bla bla}\\
+\texttt{bla bla ma in un'altra riga}
+
+\subsection{Sottosezione 1}
+Grazie al package parskip se vai a capo nel .tex lasciando una riga
+
+ti mette un po' di spazio anche nel pdf.\\
+Attenzione però che ogni tanto questa feature fa lasciare troppo spazio tra testo e immagini / tabelle, se capita prova a togliere un po' di righe vuote.
```

Comment:
Spiegherei che cambia il modo in cui vengono impaginati i `\n\n` e che è un buon trick per allungare di qualche pagina il documento ;).
```suggestion
Attenzione però che ogni tanto questa feature fa lasciare troppo spazio tra testo e immagini / tabelle, se capita prova a togliere un po' di righe vuote. Senza questo pacchetto, una doppia new line (\texttt{\n\n}) creea un nuovo paragrafo, la cui prima riga viene leggermente indentata (un comportamento indesiderato se vieni da altri strumenti di stesura). Eventualmente, si può usare per allungare di qualche pagina alla tesi, evitando di abusarne.
```